### ISSUE TITLE
Batch Size 

### ISSUE THREAD

**Issue by pranavsinghps1** [ChrisMats/medical_transformers#4]
Hello,
I was wondering what batch size was used for the supervised learning/fine-tuning task? From my understanding,
The paper mentions 256, which is for self-supervised training, and 64 (https://github.com/ChrisMats/medical_transformers/blob/ecd460a66ea2b03a83caf4c94546ef012e17e049/params.json#L99) for supervised learning across all three datasets. Please correct me if I am wrong.

Thank You.

**Comment by ChrisMats** [comment]
Hi,
You are correct. The batch size that we used for SSL was 256. 
The batch size that you define at `dataloader_params.trainloader.batch_size` is the batch size per rank (GPU). e.g. if you have 4 GPUs (controlled from `system_params.which_GPUs`) then your total batch size will be 256.

**Comment by pranavsinghps1** [comment]
Understood, Just to make it clear batch size for SSL: 256 and for the supervised learning with and without SSL, was 256 as well?


And also, it would be helpful if you could shed some light on the number of epochs used for labelled finetunning and without dino supervised learning.

Thank you.

**Comment by ChrisMats** [comment]
For the supervised training we used a batch size of 64.
The number of epochs varies from dataset to dataset and the training setting (rand init, pretrained).

**Comment by pranavsinghps1** [comment]
Thank You, and finally, the results mentioned in Table 1 are results on the Test set right?

**Comment by ChrisMats** [comment]
That is correct
